### PR TITLE
Comment Book

### PR DESCRIPTION
**Pull request by phuc021**

### CODE REVIEW

**Comment by phuc021** [comment]
ready

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+use Illuminate\Support\Facades\Auth;
+use App\Http\Requests\CommentRequest;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+
+class CommentController extends Controller
+{
+	public function __construct()
+	{
+    	$this->middleware('auth');
```
Comment (quoctd-0607) [inline app/Http/Controllers/CommentController.php:18]: indent. 4 space em

**Comment by phuc021** [comment]
ready

File: app/Http/Controllers/CommentController.php (hunk 1 above)
Comment (phuc021) [reply app/Http/Controllers/CommentController.php]: dạ e fix rồi anh

**Pull request merged**